rust/api/src/kv_store.rs

```diff
@@ -1,9 +1,12 @@
 use crate::error::VssError;
 use crate::types::{
-	DeleteObjectRequest, DeleteObjectResponse, GetObjectRequest, GetObjectResponse,
+	DeleteObjectRequest, DeleteObjectResponse, GetObjectRequest, GetObjectResponse, KeyValue,
 	ListKeyVersionsRequest, ListKeyVersionsResponse, PutObjectRequest, PutObjectResponse,
 };
 use async_trait::async_trait;
+use bytes::Bytes;
+use rand::distributions::Alphanumeric;
+use rand::{thread_rng, Rng};
```

Comment:
I'm seeing compilation errors for this. Could you properly mark anything test-related with `#[cfg(test)]`? Or if this module is entirely going to be for tests, could you mark the entire module accordingly.